pandas/tests/groupby/test_categorical.py

```diff
@@ -1332,7 +1333,8 @@ def test_series_groupby_on_2_categoricals_unobserved_zeroes_or_nans(
     # Tests whether the unobserved categories in the result contain 0 or NaN
 
     if reduction_func == "ngroup":
-        pytest.skip("ngroup is not truly a reduction")
+        # https://github.com/pytest-dev/pytest/issues/7495
+        pytest.skip("ngroup is not truly a reduction")  # type: ignore
```

Comment:
tbh here, this is only causing errors since the function is being checked since we have type annotations in the function signature.

I think our policy is that we don't type check the tests themselves.

maybe I should remove the type annotations here instead. @WillAyd

Reply:
@WillAyd because have `warn_unused_ignores = True` we need to have pytest 6.0.0 as the minimum version in the dev environment if keep these ignores. I think that removing the type annotations on these tests is probably best.

Reply:
remove type annotations from pandas/tests/groupby/test_categorical.py. will remove pytest pin from environment.yml after ci run.